lib/faker/travel/airport.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+module Faker
+  class Travel
+    class Airport < Base
+      class << self
+        ##
+        # Produces random large US Airport by name.
+        #
+        # @return [String]
+        #
+        # @example
+        # Faker::Travel::Airport.large_airport_us #=> "Los Angeles International Airport"
```

Comment:
there is some missing indentation on the documentation:
```suggestion
        #   Faker::Travel::Airport.large_airport_us #=> "Los Angeles International Airport"
```